components/schemes/Typography/tests/unit/MDCTypographySchemeTests.m

```diff
@@ -192,4 +204,75 @@ - (void)testTypographyScheme201902UseCurrentContentSizeCategoryWhenAppliedIsDisa
   XCTAssertTrue(scheme.useCurrentContentSizeCategoryWhenApplied);
 }
 
+- (void)test201902ScalesOniOS10AndUpWhenTraitEnvironmentIsProvided {
+  if (@available(iOS 10.0, *)) {
+    // Given
+    MDCTypographyScheme *schemeWithNoTraitEnvironment =
+        [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201902];
+
+    // When
+    MDCTypographySchemeMockTraitEnvironment *traitEnvironment =
+        [[MDCTypographySchemeMockTraitEnvironment alloc] init];
+    traitEnvironment.traitCollection = [UITraitCollection
+        traitCollectionWithPreferredContentSizeCategory:UIContentSizeCategoryExtraLarge];
+    MDCTypographyScheme *scheme =
+        [schemeWithNoTraitEnvironment schemeForTraitEnvironment:traitEnvironment];
+
+    // Then
+    XCTAssertGreaterThan(scheme.headline1.pointSize,
```

Comment:
Please assert greater than or equal. Scaling doesn't necessitate an increased font size (it could have a maximum).